SPEC-annex-B-elf-embedding.md

```diff
@@ -0,0 +1,29 @@
+# Annex B
+
+Annex B contains a method of embedding OmniBOR ID into elf files.
+
+### Embedded Artifact Dependency Graph Identifier
+If an artifact contains an embedded artifact dependency graph identifier, then implementations MUST conform to the file format recommendations specified in this document.
```

Comment:
I think maybe there's a consistency issue between using "OmniBOR Identifier," "GitOID," or "artifact dependency graph identifier" in different places. Do we want to standardize on a single term we use for the identifier of an artifact in OmniBOR?

Reply:
Might it make sense to rename "OmniBOR document" to "Artifact Input Document" AID.

Reply:
I like Jeremy's suggestion of the word "manifest". Perhaps "Artifact Input Manifest"?